src/script/lua_api/l_mainmenu.cpp

```diff
@@ -594,31 +594,18 @@ int ModApiMainMenu::l_create_world(lua_State *L)
 /******************************************************************************/
 int ModApiMainMenu::l_delete_world(lua_State *L)
 {
-	int worldidx	= luaL_checkinteger(L,1) -1;
-
+	int world_id = luaL_checkinteger(L,1) - 1;
```

Comment:
spacing around `,`

Reply:
Isn't spacing required only to the _right_ of `,`?

Reply:
Yes. Space *after* `,`

Reply:
Thank you for clarifying

Reply:
added